src/internal/operators/scan.ts

```diff
@@ -45,14 +45,14 @@ export function scan<T, R>(accumulator: (acc: R, value: T, index: number) => R):
  * @see {@link mergeScan}
  * @see {@link reduce}
  *
- * @param {function(acc: R, value: T, index: number): R} accumulator
+ * @param {function(acc: A, value: V, index: number): A} accumulator
  * The accumulator function called on each source value.
- * @param {T|R} [seed] The initial accumulation value.
- * @return {Observable<R>} An observable of the accumulated values.
+ * @param {V|A} [seed] The initial accumulation value.
```

Comment:
https://github.com/ReactiveX/rxjs/pull/4858/files#diff-bcface33933d00ae116bcf19e16b2b7cR55 specifies seed as `S` but it doesn't extends `V|A` as comment says - maybe this needs to be S?

Reply:
uh welp maybe above overloading does its job

Reply:
Yeah. These docs need an overhaul, probably.